system/src/jvmMain/kotlin/com/splendo/kaluga/system/network/NetworkManager.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ Copyright 2020 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.system.network
+
+actual class NetworkManager actual constructor(networkStateRepo: NetworkStateRepo, context: Any?) : BaseNetworkManager(networkStateRepo) {
```

Comment:
Use a Builder to pass platform context. You can also use `ApplicationHolder`, probably location is the best example to look at.